### PR TITLE
Enforce monospace for code blocks

### LINKED ISSUE #386: New layout: `code` sections are not using monospaced font
E.g. in ["Credits"](http://jrudolph.github.io/akka.github.com/blog/news/2017/05/02/akka-2.4.18-released.html) section of blog posts:

screenshot from 2017-05-09 18 11 16

That's because style.css starts with this:

```
html,body,div,span,applet,object,iframe,h1,h2,h3,h4,h5,h6,p,blockquote,pre,a,abbr,acronym,address,big,cite,code,del,dfn,em,img,ins,kbd,q,s,samp,small,strike,strong,sub,sup,tt,var,b,u,i,center,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,aside,canvas,details,embed,figure,figcaption,footer,header,hgroup,menu,nav,output,ruby,section,summary,time,mark,audio,video {
    margin: 0;
    padding: 0;
    border: 0;
    font-size: 100%;
    font: inherit;
    vertical-align: baseline;
}
```

which overrides any other font styles for all the listed elements. Not sure what the underlying problem is (I don't know about the css creation pipeline).

### PR DESCRIPTION
**Pull request by johanandren**
Fixes #386

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +23,6 @@ After cloning, cd into the `akka/akka.github.com` directory and run:
 
 To see the generated site, just visit `http://localhost:4000`.
 
-To watch for changes:
-
-    jekyll serve --baseurl '' --watch
-
```
Comment (johanandren) [inline README.md]: The above instruction already watches for changes so this was superfluous

**Comment by johanandren** [comment]
Not quite sure about the relation between the page.css and the sass files, will ask the layout-consultant.

**Comment by johanandren** [comment]
Ok, the css files are essentially old leftovers, I'm removing them all and fixing this in the sass files

**Comment by johanandren** [comment]
Running for review on my gh-pages: http://johanandren.github.io/akka.github.com/blog/

**Comment by johanandren** [comment]
I'll just go ahead and merge this.

**Comment by jrudolph** [comment]
Thanks for the cleanup, that'll help others a lot to figure out where to make changes.

**Pull request merged**